tests/phpunit/BoltListener.php

```diff
@@ -11,7 +17,7 @@
  *
  * @author Gawain Lynch <gawain.lynch@gmail.com>
  */
-class BoltListener implements \PHPUnit_Framework_TestListener
+class BoltListener implements TestListener
```

Comment:
Since you are here, what do you think about extending `BaseTestListener` instead? That way we don't have to have all these empty method bodies.